doc/release/0.2.rst

```diff
@@ -2,57 +2,65 @@
 #. Split 3 classes in ``database3.py`` into 3 files (`PR#955 <https://github.com/terrapower/armi/pull/955>`_)
-#. savePhysicsFiles setting to copy physics kernel input and output to directories organized by cycle and time step number (e.g., c2n1) to retain files from a simulation and avoid overwriting them. (`PR#958 <https://github.com/terrapower/armi/pull/952>`_)
-#. Add pinQuantities parameter category for blockParams that have spatial distribution within a block.
-#. Use r.core.p.axialMesh instead of r.core.refAssem.getAxialMesh() for the uniform mesh converter. (`PR#959 <https://github.com/terrapower/armi/pull/959>`_)
 #. Split algorithms specific to hex assemblies out of ``FuelHandler``. (`PR#962 <https://github.com/terrapower/armi/pull/962>`_)
-#. Add ability to load from a db using negative node index
-#. Add group structures for 21- and 94-groups used in photon transport
-#. Fix numerical diffusion bug in uniform mesh converter that affects number densities and cumulative parameters like DPA. (`PR#992 <https://github.com/terrapower/armi/pull/992>`_)
-#. A more complete set of nuclides has been added to the framework to decouple the optional loading of RIPL-3 data and to embed 4614 nuclides into the standard loading process for the framework. This allows for further extensibility of framework to support end-users. (`PR#998 <https://github.com/terrapower/armi/pull/998>`_)
-#. Add block parameter, fuelCladLocked, to track whether or not the fuel and clad are locked. (`PR#1038 <https://github.com/terrapower/armi/pull/1038>`_)
-#. An explicit fission product modeling option was added in `PR#1022 <https://github.com/terrapower/armi/pull/1022>`_.
-#. Cleanup of unnecessary and confusing functionality in axialExpansionChanger.py. (`PR#1032 <https://github.com/terrapower/armi/pull/1032>`_)
-#. Axially expand from cold to hot before deepcopy of assemblies into reactor improving speed and preserving same functionality. (`PR#1047 <https://github.com/terrapower/armi/pull/1047>`_)
-#. Add a how-to on restart calculations in the docs
-#. Overhaul of the tight coupling routine in ARMI. Removal of ``looseCoupling`` setting and introduction of new tight coupling user interface. (`https://github.com/terrapower/armi/pull/1033`_)
-#. General improvements to efficiency in many places to speed up uniform mesh conversion. (`PR#1042 <https://github.com/terrapower/armi/pull/1042>`_)
+#. Added 4614 nuclides to decouple the loading of RIPL-3 data from the standard framework run. (`PR#998 <https://github.com/terrapower/armi/pull/998>`_)
+#. Overhaul of the tight coupling routine in ARMI, and removal of ``looseCoupling`` setting. (`https://github.com/terrapower/armi/pull/1033`_)
+#. Added ``savePhysicsFiles`` setting to copy physics kernel I/O to directories organized by cycle and time step (e.g., c2n1). (`PR#958 <https://github.com/terrapower/armi/pull/952>`_)
```

Comment:
The PR hyperlink and tag(?) don't match here. 

@mgjarrett was this one of your PRs?

Reply:
good catch!

Reply:
```suggestion
#. Added ``savePhysicsFiles`` setting to copy physics kernel I/O to directories organized by cycle and time step (e.g., c2n1). (`PR#952 <https://github.com/terrapower/armi/pull/952>`_)
```